roles/apache/tasks/configuration.yml

```
@@ -0,0 +1,43 @@
+---
+
+- name: Add httpd configuration file
+  copy:
+    src: httpd.conf
+    dest: /etc/httpd/conf/httpd.conf
+    mode: '0644'
+    backup: yes
+
+- name: Add ssl configuration file
+  copy:
+    src: ssl.conf
+    dest: /etc/httpd/conf.d/ssl.conf
+    mode: '0644'
+    backup: yes
+
+- name: Add strong_ssl configuration file
+  copy:
+    src: strong_ssl.conf
+    dest: /etc/httpd/conf.d/strong_ssl.conf
+    mode: '0644'
+    backup: yes
+
+- name: Check if chain file exists
+  become: no
+  local_action:
+    module: stat
+    path: private_files/{{ inventory_hostname }}/chain-{{ inventory_hostname }}.pem
+  register: chain_file
+
+- name: Configure Virtualhost
+  template:
+    src: virtualhost.j2
+    dest: /etc/httpd/conf.d/{{ inventory_hostname }}-host.conf
```

This may not be necessary and in the end I think we have more beautiful results on remote machines.

```suggestion
    dest: /etc/httpd/conf.d/{{ inventory_hostname }}.conf
```

---

done